lib/ansible/modules/lineinfile.py

```diff
@@ -480,6 +533,9 @@ def absent(module, dest, regexp, line, backup):
     def matcher(b_cur_line):
         if regexp is not None:
             match_found = bre_c.search(b_cur_line)
+        elif literal is not None:
+            match_found = to_bytes(
+                literal, errors='surrogate_or_strict') in b_cur_line
```

Comment:
No reason to break this line:
```suggestion
            match_found = to_bytes(literal, errors='surrogate_or_strict') in b_cur_line
```